imports/core/startup/client/routes.js

```diff
@@ -1,5 +1,8 @@
-import VotingPage from '../../../voting/ui/pages/VotingPage.vue';
+import VotingPage from '/imports/voting/ui/pages/VotingPage'
+import SingleProposal from '/imports/voting/ui/pages/SingleProposal'
 
 export default [
   { path: '/', name: 'voting', component: VotingPage },
+  { path: '/sorting/popular', name: 'popular', component: VotingPage },
```

Comment:
I propose to handle sorting type as path param (like `'/sorting/:filter'`).

Reply:
@davidyuk excellent you are a man !!